addons/project_issue/project_dashboard.xml

```diff
@@ -8,7 +8,7 @@
                 <field name="use_issues"/>
                 <field name="issue_ids" invisible="1"/>
             </field>
-            <xpath expr="//div[contains(@class, 'o_project_kanban_boxes')]" position="inside">
+            <xpath expr="//a[contains(@class, 'o_first')]" position="after">
```

Comment:
By doing this 

..//span[contains(@class, 'o_label')][1]

If the tasks are not enabled, the 'Issue' block is not displayed, as it's not in the dom